tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -182,7 +189,7 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
   dataSeries$?: Observable<ScalarCardDataSeries[]>;
   chartMetadataMap$?: Observable<ScalarCardSeriesMetadataMap>;
   linkedTimeSelection$?: Observable<TimeSelectionView | null>;
-  stepSelectorTimeSelection$?: Observable<TimeSelection | null>;
+  stepSelectorTimeSelection$ = new BehaviorSubject<TimeSelection | null>(null);
```

Comment:
I think this should be moved out of this list of declarations.

Reply:
I moved this to be with the selectors and made it `readonly` as I think it is doing the same thing in spirit. LMK if you'd rather it live someplace else, there isn't a ton of precedent for this.